src/main/scala/nl/knaw/dans/lib/logging/servlet/masked/response/MaskedSetCookie.scala

```diff
@@ -21,18 +21,18 @@ import org.scalatra.ScalatraBase
 private[masked] trait MaskedSetCookie extends ResponseLogExtensionBase {
   this: ScalatraBase =>
 
-  abstract override def formatResponseHeader(header: HeaderMapEntry): HeaderMapEntry = {
+  abstract override protected def formatResponseHeader(header: HeaderMapEntry): HeaderMapEntry = {
     super.formatResponseHeader(header) match {
       case (name, values) if name.toLowerCase == "set-cookie" =>
-        name -> values.map(formatCookieValue)
+        name -> values.map(formatCookieHeader)
       case otherwise => otherwise
     }
   }
 
-  private def formatCookieValue(value: String): String = {
+  private def formatCookieHeader(value: String): String = {
```

Comment:
You'll need the same method for the header "cookie" for requests. Aren't you duplicating code?

Reply:
good point; as discussed via Skype, I moved the masking logic to a separate module. As a bonus, this can now all easier be unit tested and code duplication can be avoided as much as possible.